migrations/task_create_time.go

```diff
@@ -0,0 +1,101 @@
+package migrations
+
+import (
+	"time"
+
+	"github.com/evergreen-ci/evergreen"
+	"github.com/mongodb/anser"
+	"github.com/mongodb/anser/db"
+	"github.com/mongodb/anser/model"
+	"github.com/pkg/errors"
+	"gopkg.in/mgo.v2/bson"
+)
+
+const (
+	migrationTaskCreateTime = "task-create-time"
+	versionsCollection      = "versions"
+)
+
+type versionInfo struct {
+	CreateTime time.Time `bson:"create_time"`
+}
```

Comment:
I'd move this into an anonymous structure in the scope of the migration

Reply:
Good idea, done.